modules/renderer/src/text-measurer.ts

```diff
@@ -34,19 +34,19 @@ export default class TextMeasurer {
     this.ctx.font = `${fontStyle} ${fontWeight} ${fontSize}px ${fontFamily}`
   }
 
-  measure(text: string, fontSize: number, fontFamily: string, fontStyle: string): Size {
+  measure(text: string, _fontSize: number, _fontFamily: string, _fontStyle: string): Size {
```

Comment:
Can you make the second parameter a `TextMeasurerOptions`

Reply:
Will do!